src/applications/applications.service.ts

```
@@ -1,49 +1,53 @@
    */
   async applyForJob(
     jobId: string,
     freelancerId: string,
     recruiterId: string,
+    coverLetter: string,
   ): Promise<Application> {
-    // Input validation
-    if (!jobId || !freelancerId || !recruiterId) {
-      throw new Error('All parameters (jobId, freelancerId, recruiterId) are required');
+    if (!jobId || !freelancerId || !recruiterId || !coverLetter) {
+      throw new ConflictException('All parameters (jobId, freelancerId, recruiterId, coverLetter) are required');
```

ESLint

[error] Replace `'All·parameters·(jobId,·freelancerId,·recruiterId,·coverLetter)·are·required'` with `⏎········'All·parameters·(jobId,·freelancerId,·recruiterId,·coverLetter)·are·required',⏎······`

(prettier/prettier)